### ISSUE TITLE
A brief readme to install and run the tool

### ISSUE THREAD

**Issue by crazyhottommy** [ljdursi/mergevcf#2]
Hi,

Following your [blog post](http://simpsonlab.github.io/2015/06/15/merging-sv-calls/)
Thanks very much for making this tool available. Could you please update the readme a bit so that I can use this tool?

Thanks,
Ming

**Comment by ljdursi** [comment]
Yes, it's probably time for me to write some documentation, and maybe put it on pypi.  What is the highest priority for you - a quickstart, something like that?

**Comment by crazyhottommy** [comment]
I want to try it out. One caveat is that sv vcf files from different tools may vary differently.
manta output vcf4.1 https://github.com/Illumina/manta/blob/master/src/markdown/mantaUserGuide.md

> it can sometimes be convenient to express structural variants in BEDPE format. For such applications we recommend the script vcfToBedpe available from:

https://github.com/ctsa/svtools

> This repository is forked from @hall-lab with edits to support VCF 4.1 SV format and match Manta's portability contstaints.

lumpy output vcf4.2 https://github.com/arq5x/lumpy-sv
the author also has a tool for merging which I have not tried out yet https://github.com/hall-lab/lumpy-merge

I would like to know the details of the merging process.

Thank you!
